packages/map-template/src/components/Map/Map.jsx

```diff
@@ -95,9 +97,9 @@ function Map({ onLocationClick, onVenueChangedOnMap }) {
     useEffect(() => {
         if (mapsIndoorsInstance) {
             if (filteredLocations) {
-                mapsIndoorsInstance.filter(filteredLocations.map(location => location.id));
+                mapsIndoorsInstance.highlight(filteredLocations.map(location => location.id));
```

Comment:
If a customer is using another JS SDK version (we allow that - we only load the JS SDK if the `mapsindoors` namespace doesn't exist), this will break. Should we handle this better?

Reply:
Added a check if the method exists in the SDK. ✅